RadixWallet/Features/TransactionReviewFeature/TransactionReviewProofs/TransactionReviewProofs+View.swift

```diff
@@ -29,42 +29,15 @@ extension TransactionReviewProofs {
 					.padding(.bottom, .medium2)
 
 					ForEach(viewStore.proofs) { proof in
-						VStack(spacing: 0) {
-							let metadata = proof.metadata
-							ProofView(thumbnail: metadata.iconURL, name: metadata.name) {
-								viewStore.send(.proofTapped(id: proof.id))
+						VStack(spacing: .medium3) {
+							ResourceBalanceView(proof.resourceBalance.viewState, appearance: .compact) {
```

Comment:
You are adding a `VStack` for each proof, which is pointless. Need to revert the order, as shown in previous [comment](https://github.com/radixdlt/babylon-wallet-ios/pull/1214/#discussion_r1679200536)

Reply:
Oh, right!